pages/search/search.jsx

```diff
@@ -58,22 +80,77 @@ export default React.createClass({
       // In the meanwhile, we have to rely on document.querySelector(`#search-box`) to trigger input's focus() function
       document.querySelector(`#search-box`).focus();
     });
-  },
+  }
+
+  renderSearchBar() {
+    return <div className={classNames({activated: true, 'search-bar': true, 'mb-2': this.props.moderation})}>
+            <DebounceInput id="search-box"
```

Comment:
why not indent it like you indent https://github.com/mozilla/network-pulse/pull/575/files#diff-0634d7511c187586b619180db9c55f90R121 ? `id="..."` on the next line, indented, and `/>` on its own line at the same indent as the `<DebounceInput` opening